tesseract_task_composer/cmake/tesseract_task_composer-config.cmake.in

```diff
@@ -0,0 +1,39 @@
+@PACKAGE_INIT@
```

Comment:
Doesn't the current version of RICB take care of this without having to create a separate file?

Reply:
I just followed how things were done in tesseract_kinematics, though I am not sure it supports conditional dependencies.